app/client/src/store/actions.ts

```diff
@@ -61,4 +61,40 @@ export default {
       commit('setStatus', { task: 'network', data: 'submitted' });
     });
   },
+  async getStatus({ commit, state }: { commit: Commit, state: RootState }) {
+    await axios.post(
+      `${config.server_url}/status`,
+      { hash: state.projectHash },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    ).then((response) => {
+      const prev = { ...state.analysisStatus };
```

Comment:
I don't think you really need to do this expansion here, doesn't look like you're editing `prev` at all.